src/inverseRobot.js

```diff
@@ -7,7 +7,24 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const objArr = Object.entries(robot);
+  const reverseArr = [];
+  const values = Object.values(robot);
+  const isDuplicate = values.some((item, index) =>
+    index !== values.indexOf(item)
+  );
```

Comment:
lets fill new object with needed props
```suggestion
const newRobot = {};
```